src/uptane/initialize.cc

```diff
@@ -114,6 +128,8 @@ InitRetCode Repository::initTlsCreds(const ProvisionConfig& provision_config) {
     LOGGER_LOG(LVL_trace, "Device already registered, proceeding");
     // set provisioned credentials
     http.setCerts(ca, cert, pkey);
+    director.setTlsCreds(ca, cert, pkey);
+    image.setTlsCreds(ca, cert, pkey);
```

Comment:
Is this right?  I think we now only construct a single implementation of HttpInterface and pass down from Aktualizr::run().  The TufRepository::setTlsCreds() member function should probably go too, because TufRepository has no business messing with something that it has been given :)